app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -458,6 +458,9 @@ class StatePlayerRecyclerViewAssembler private constructor(
   /**
    * Shows a celebratory animation with a congratulations message and confetti when the learner submits
    * a correct answer.
+   *
+   * @param feedback Oppia's feedback to the learner's most recent answer. If the feedback is empty,
+   * talkback confirms correct answer by announcing "Correct" banner.
```

Comment:
```suggestion
   *     talkback confirms correct answers by a separate announcement.
```

Updated to not be overly specific to the implementation, and to indent it per https://developer.android.com/kotlin/style-guide#block_tags.